src/utils/services/index.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+export async function getVideos(keyword) {
```

Comment:
I would suggest moving the base URL outside the functions. Also, you could declare some variables at the top to make them more visible, if in the future you need to update them you can do it more easily.

```javascript
const baseUrl = 'https://www.googleapis.com/youtube/v3';
const apiKey = process.env.REACT_APP_API_KEY;
const maxResults = 25;

function getVideos(keyword) {
  const searchUrl = `${baseUrl}/search?maxResults=${maxResults}&key=${apiKey}&part=snippet&q=${keyword}`
  // rest of the function
}

function getVideoDetails(videoId) {
  const videosUrl = `${baseUrl}/videos?id=${videoId}&key=${apiKey}&part=snippet`
  // rest of the function
}
```